src/Components/Misc/Hat.tsx

```diff
@@ -23,16 +23,31 @@ export const Hat: FC<Props> = ({linkID}) => {
 
   const {r, g, b, l} = colorFromId(linkID);
   const colors = {
-    backgroundColor: `rgb(${r}, ${g}, ${b})`,
-    // color: (l <= 50)? 'white' : 'black'
+    backgroundColor: `rgb(${r}, ${g}, ${b})`
   }
 
+  const scrollToDot = (e: any) => {
+    const target = document.querySelector(`div.dot[link-id="${linkID}"]`);
+    if (target !== null) {
+      target.scrollIntoView({behavior: 'smooth', block: 'center'});
+      let isWhite = false;
+      const targetContainer = target.parentElement?.parentElement;
+      const flash = setInterval(() => {
+        if (isWhite) targetContainer?.classList.remove('flash-on');
+        else targetContainer?.classList.add('flash-on');
+        isWhite = !isWhite
+      }, 250);
```

Comment:
We want to make sure that the flash is set to off at the very end here. Right now it's possible the Interval ends with the flash set to on.

Reply:
The set timeout will now set the flash to off when it cancels the interval